Teme/Ubungen/BookRent/src/main/java/Audiobook.java

```diff
@@ -0,0 +1,16 @@
+import javax.swing.*;
+import java.awt.*;
+
+public class Audiobook extends Book {
+    static Component frame;
```

Comment:
You should not add the visual integration here. The class should have strictly what an Audiobook has